docs/operating/configuration.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Variable Configuration
+proofedDate: na
+iterationBy: na
+includedInSite: true
+approvedBy: na
+comment: Do the API calls belong here too? https://www.notion.so/neonlabs/Neon-Specific-API-methods-3402baaad8fa4daeb12642495cf85eb3
```

Comment:
@afalaleev I assume the API calls need to be on a different page?

Reply:
Yes, it will another page.